src/tokio.rs

```diff
@@ -256,6 +354,24 @@ mod tests {
         Ok(())
     }
 
+    #[tokio::test]
+    async fn test_receive_can_fd() -> Result<()> {
```

Comment:
Actually, and tests interacting with _vcan0_ should be behind the "vcan_tests" feature, and all need to be serialized to prevent interactions. But I can add that.